deployment_eks/.specific/architecture.adoc

```diff
@@ -10,20 +10,20 @@ AWS Cloud.
 .Quick Start architecture for {partner-product-short-name} on AWS
 image::../docs/deployment_guide/images/architecture_diagram.png[Architecture]
```

Comment:
I think this directory path needs to have `/docs` removed. That's what works for the cfn version of the guide.

Reply:
No, `/docs/ `need to remain here. Do not change this.